### PR TITLE
Update container-selinux.8 man page

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

File: container.te [hunk 1]
```diff
@@ -1476,6 +1476,7 @@ optional_policy(`
 	unconfined_domain(kubelet_t)
 ')
 
+manage_chr_files_pattern(container_engine_t, fusefs_t, fusefs_t)
```
Comment (lsm5) [inline container.te:1479]: is this a dup? I see the same line already on 1459

File: container_selinux.8 [hunk 2]
```diff
@@ -312,29 +260,13 @@ container policy stores data with multiple different file context types under th
...
 .PP
 .B STANDARD FILE CONTEXT
 
 SELinux defines the file context types for the container, if you wanted to
 store files with these types in a different paths, you need to execute the semanage command to specify alternate labeling and then use restorecon to put the labels on disk.
 
-.B semanage fcontext -a -t container_ro_file_t '/srv/mycontainer_content(/.*)?'
+.B semanage fcontext -a -t container_var_lib_t '/srv/container/content(/.*)?'
 .br
 .B restorecon -R -v /srv/mycontainer_content
```
Comment (lsm5) [inline container_selinux.8:271]: should this also change to `/src/container/content` ?

**Comment by lsm5** [comment]
ignore centos stream 9 failure.

LGTM overall apart from the couple of comments. 

Suggestion: Maybe we should consider rewriting the manpage in markdown and have go-md2man or other such tool generate the actual manpage at build time ?

File: container_selinux.8 (hunk 2 above)
Comment (rhatdan) [reply container_selinux.8:271]: Yes this looks like a bug in the tool used to generate the man page.

**Comment by rhatdan** [comment]
These man pages are autogenerated, based on policy changes and the overall policy on the system.

**Pull request merged**